repo/fsrepo/fsrepo.go

```diff
@@ -185,13 +197,19 @@ func open(repoPath string) (repo.Repo, error) {
 	return r, nil
 }
 
-func newFSRepo(rpath string) (*FSRepo, error) {
+func newFSRepo(rpath string, userConfigFilePath string) (*FSRepo, error) {
 	expPath, err := homedir.Expand(filepath.Clean(rpath))
 	if err != nil {
 		return nil, err
 	}
 
-	return &FSRepo{path: expPath}, nil
+	configFilePath, err := config.Filename(rpath, userConfigFilePath)
+	if err != nil {
+		// FIXME: Personalize this when the user config path is "".
```

Comment:
Do we need to do this?

Reply:
Not blocking, just a normal FIXME, can remove if you want.